src/utils/normalization.ts

```diff
@@ -78,14 +78,16 @@ export async function disambiguateReleaseParam(
 /**
  * Convert to lowercase if looks like slug
  */
-export async function lowercaseIfSlug(s: string) {
-	return s.includes('/') ? s.toLowerCase() : s;
+export function lowercaseIfSlug(s: string) {
+	return s.includes('/')
+		? Promise.resolve(s.toLowerCase())
+		: Promise.resolve(s);
```

Comment:
What is the purpose of making these `Promise.resolve` instead of just letting the function return the strings?